### DISCUSSION TITLE
Inline tool for multiple blocks

### DISCUSSION THREAD

**Discussion by myktsk** [codex-team/editor.js#2077] [Ideas]
Hey 👋
Would it be possible to enable control over multiple blocks?

For example, 
- make texts inside multiple blocks bold
- convert multiple blocks into lists

#1942 is similar but I mean more specifically for the inline tools.

p.s. Thank you for your amazing work!

**Comment by MussaratAziz** [comment]
@myktsk @n0str  any update?

**Comment by jamesfebin** [comment]
I want to implement this myself, but I am stuck in figuring out how multiple selections of block events can be listened to. 

How do I listen to multiple block select?
